src/hera/task.py

```diff
@@ -172,9 +172,14 @@ class Task:
     command: Optional[List[str]] = None
         The command to use in the environment where the function runs in order to run the specific function. Note that
         the specified function is parsed, stored as a string, and ultimately placed in a separate file inside the task
-        and invoked via `python script_file.py`. This command offers users the opportunity to start up the script in a
-        different way e.g `time python` to time execution, `horovodrun -p X` to use horovod from an image that allows
-        training models on multiple GPUs, etc.
+        and invoked via `python script_file.py`. Also, note that when neither command nor args are set, the command
+        will default to python. This command offers users the opportunity to start up the script in a different way
+        e.g `time python` to time execution, `horovodrun -p X` to use horovod from an image that allows training models
+        on multiple GPUs, etc.
+    args: Optionals[List[str]] = None
+        Optional list of arguments to run in the container. This can be used as an alternative to command, with the
+        advantage of not overriding the set entrypoint of the container. Note that when argo is using the emissary
```

Comment:
This docstring seems incomplete? Also, perhaps we can add a `Notes` section for the emissary executor note?
```suggestion
        advantage of not overriding the set entrypoint of the container. Note that when argo is using the emissary...
```

Reply:
Ive added some more info - would be good to get feedback on it